src/gcp_scanner/scanner.py

```diff
@@ -269,7 +270,13 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
 
       sa_results_data = json.dumps(sa_results, indent=2, sort_keys=False)
 
-      with open(out_dir + '/%s.json' % project_id, 'a',
+      # Generate current timestamp to append to output filename
+      scan_time_suffix = datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+      
+      # Generate the output filename and append the timestamp (to make filename unique for every scan)
+      output_file_name = f'{out_dir}/{project_id}-{scan_time_suffix}.json'
```

Comment:
- rename to output_path
- use pathlib to combine paths

Reply:
Great idea :slightly_smiling_face: 
Will make sure to use pathlib in future for all file handling tasks :)